examples/iris/iris.rb

```diff
@@ -31,10 +31,10 @@ def fwd(x)
 optimizer.setup(model)
 
 iris = Datasets::Iris.new
-x = iris.each.map {|r| r.each.to_a[0..3]}
+x = iris.to_table.fetch_values(:sepal_length, :sepal_width, :petal_length, :petal_width).transpose
 
 # target
-y_class = iris.each.map {|r| r.class}
+y_class, = iris.to_table.fetch_values(:class)
```

Comment:
We don't need to use `fetch_values` to get only one column value. We can just use `iris_table[:class]`.